rcloud.support/R/rcloud.support.R

```diff
@@ -407,14 +410,17 @@ rcloud.fork.notebook <- function(id, source = NULL) {
                                      list(owner=owner,
                                           description=src.nb$content$description,
                                           id=src.nb$content$id))
-    } else ## src=dst, regular fork
+    } else {## src=dst, regular fork
         new.nb <- fork.gist(id, ctx = src.ctx)
-
+        rcloud.config.set.recently.modified.notebook(id, new.nb$content$updated_at)
```

Comment:
This is creating an entry for the *source* notebook with the updated date of the *fork*. I'll open a new issue for this.